x-pack/plugins/observability_solution/apm/server/routes/alerts/rule_types/transaction_duration/get_transaction_duration_chart_preview.ts

```diff
@@ -90,21 +90,21 @@ export async function getTransactionDurationChartPreview({
   const allGroupByFields = getAllGroupByFields(ApmRuleType.TransactionDuration, groupByFields);
 
   const aggs = {
-    timeseries: {
-      date_histogram: {
-        field: '@timestamp',
-        fixed_interval: interval,
-        min_doc_count: 0,
-        extended_bounds: {
-          min: start,
-          max: end,
-        },
+    series: {
+      multi_terms: {
+        terms: [...getGroupByTerms(allGroupByFields)],
```

Comment:
Isn't it the same as:
```suggestion
        terms: getGroupByTerms(allGroupByFields),
```

Reply:
updated in 9f0566611bb5d92b880684b8a69f0fd50a9225bd